docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -87,53 +53,11 @@ GRANT ALL ON `%`.* TO 'alice'@'%';
 See [Configuring MySQL to Use Encrypted Connections](https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html)
 in MySQL documentation for more details.
 
-## Configure Teleport
-
-Teleport Database Access is available starting from `6.0` release.
-
-Download the appropriate version of Teleport for your platform from
-our [downloads page](https://goteleport.com/teleport/download).
-
-Follow the installation [instructions](../../installation.mdx).
-
-### Start Auth/Proxy Service
-
-Create a configuration file for a Teleport service that will be running
-auth and proxy servers:
-
-```yaml
-teleport:
-  data_dir: /var/lib/teleport
-  nodename: test
-auth_service:
-  enabled: "yes"
-proxy_service:
-  enabled: "yes"
-  # Set public address proxy will be reachable at.
-  public_addr: teleport.example.com:3080
-  # MySQL proxy is listening on a separate port and needs to be enabled
-  # on the proxy server.
-  mysql_listen_addr: 0.0.0.0:3036
-ssh_service:
-  enabled: "no"
-```
-
-Start the service:
-
-```sh
-$ teleport start --config=/path/to/teleport.yaml
-```
+## Setup Teleport Auth and Proxy Services
 
-Generate a short-lived join token for the database service and save it for
-example in `/tmp/token`:
+Teleport Database Access for MySQL is available starting from `6.0` release.
 
-```sh
-$ tctl tokens add \
-    --type=db \
-    --db-name=test \
-    --db-protocol=mysql \
-    --db-uri=postgres.example.com:5432
-```
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
 ### Create Role and User
```

Comment:
Per the comment above, we could probably remove this section if we moved the include to the beginning. What do you think? 

We'd repeat this for both of the self-hosted articles.

Reply:
Replied above, these can't go at the beginning because these are a part of Teleport setup.